shiny/ui/_input_select.py

```diff
@@ -54,6 +56,8 @@ def input_selectize(
     selected: Optional[str | list[str]] = None,
     multiple: bool = False,
     width: Optional[str] = None,
+    remove_button: bool = True,
```

Comment:
I know we talked about changing this to `False`, but what about `None` or `MISSING` (and have that resolve to `True` when `multiple=True` and `False` otherwise)?

Reply:
Good idea let's go with that.